docs/tutorial/table_lookups.md

```diff
@@ -172,3 +178,47 @@ The function is first traced into:
 {% hint style="info" %}
 Fusing makes the code more readable and easier to modify, so try to utilize it over manual `LookupTable`s as much as possible.
 {% endhint %}
+
+## Using automatically created table lookup
+
+We refer the users to [this page](tutorial/extensions.md) for explanations about
+`fhe.univariate(function)` and `fhe.multivariate(function)` features, which are convenient ways to
+use automatically created table lookup.
+
+## Table lookup exactness
+
+TLUs are performed with an FHE operation called `Programmable Bootstrapping` (PBS). PBS's have a certain probability of error, which, when triggered, result in inaccurate results.
```

Comment:
Sounds a bit convoluted. Maybe "PBS's have a certain probability of error and thus may result in inaccurate results upon invocation."?